core/domain/exp_domain.py

```diff
@@ -2398,8 +2419,8 @@ def _add_id_and_proto_size_in_bytes_to_dict(
 
         # The constructor calculates the proto_size_in_bytes and adds it to the
         # domain object. So, we add that value directly to the dict.
-        exploration_dict['proto_size_in_bytes'] = (
-            exploration.proto_size_in_bytes)
+        exploration_dict['_cached_android_proto_size_in_bytes'] = (
```

Comment:
exploration_dict should have a proto_size_in_bytes attribute. The _cached_android_proto_size_in_bytes should _only_ be a private member variable within the Exploration domain object class.

Reply:
Fixed